content/dev/code/admin_sanitization.md

```diff
@@ -143,7 +143,7 @@ Zen Cart defines the following default case-insensitive sanitizers:
 
     POST values only 
     
-    uses `[^0-9a-z.!@#$%^& ()`_+-~/\\]` regex
+    uses ``[^0-9a-z.!@#$%^& ()`_+-~/\\]`` regex
```

Comment:
Why the change to double back-ticks?

Reply:
I was surprised by this too but when I looked at it in the preview, it looked the same.  @brittainmark is this needed or should it be removed here and in your other PR?